src/cr/cube/matrix.py

```diff
@@ -2616,13 +2620,14 @@ def __init__(
             table_std_err,
             column_index,
         )
-        self._all_unweighted_counts = unweighted_counts
-        self._all_counts = counts
+        self._both_counts = counts
+        self._both_unweighted_counts = unweighted_counts
```

Comment:
I don't know if `both` could be a meaningful name ... if i got the point here, all counts are referring to the complete cases for the MR. So what about `complete_counts`?

Reply:
@scanny I agree with @ernestoarbitrio here, I don't have _any_ context for the word "both" here. If you insist on using that word, please provide at least an explanation as to what it refers to. Maybe in a docstring or something.